static/assets/tutorials/node-metrics/substrate-node-template-metrics.json

```diff
@@ -517,18 +517,7 @@
     "to": "now"
   },
   "timepicker": {
-    "refresh_intervals": [
```

Comment:
this is the expected prettier behavior, we can ignore .json files if we wish, or use something else to format.

Reply:
going to ignore these in docs formatting and the `static` files